app/views/homepages/index.html.erb

```diff
@@ -0,0 +1,37 @@
+<article>
+  <h1> Media Spotlight</h1>
+    <%= link_to @media_spotlight.title, work_path(@media_spotlight.id) %>
+
+<div class="media_body">
+  <article class = "media_column">
```

Comment:
Very small note: HTML convention is to not space that out so just `class="media_column"`